packages/product-catalogue/src/components/ProductForm/index.tsx

```diff
@@ -147,37 +168,43 @@ const ProductForm = (props: ProductFormProps) => {
   }
 
   return (
-    <div className="form-container">
+    <div className="product-form form-container">
       <Formik
         initialValues={initialValues}
         validationSchema={ProductFormValidationSchema}
-        onSubmit={(values) => {
+        onSubmit={(values, actions) => {
           const payload = { ...values };
           if (isEditMode) {
             putProduct(baseURL, payload)
               .then(() => {
                 sendSuccessNotification('Successfully Updated');
```

Comment:
Do we want this to come from some constants/lang module?

Reply:
moved the strings to a lang module, though at the moment they are just that, strings without any form of operation being done on them,